lc-hax/Scripts/Commands/Host/LevelEnemiesCommand.cs

```diff
@@ -0,0 +1,19 @@
+using Hax;
+using System;
+using UnityEngine;
+
+[DebugCommand("/enemies")]
+public class LevelEnemiesCommand : ICommand {
+
+    public void Execute(StringArray _) {
+        if (Helper.RoundManager == null) return;
+        if (Helper.RoundManager.currentLevel == null) return;
+        string Msg = "";
+        foreach (System.Collections.Generic.KeyValuePair<string, GameObject> enemy in Helper.SpawnableEnemies) {
+            Msg += $"{enemy.Key}, ";
+        }
+        Console.Write(Msg);
```

Comment:
There's a lot of leftover debug prints so I'll have to be even more careful reviewing this.

Reply:
The Console.Write will just print all spawnable enemies to copy them in case Bepinex is open and running.

Reply:
Why do we need to copy them?

Reply:
Just in case